### ISSUE TITLE
Eliminate moz-focus-inner border on buttons

### ISSUE THREAD

**Issue by albell** [necolas/normalize.css#393]
Please note that I am _not_ talking about disabling the standard default CSS outline. This is something else, a dotted black line _inside_ the element that is connected to focus but is _not_ the focus outline itself. See <a href="http://stackoverflow.com/questions/71074/how-to-remove-firefoxs-dotted-outline-on-buttons-as-well-as-links/199319#199319">Stack Overflow</a> discussion. No other browser does this. It serves no accessibility function. AFAIAC it is an ugly relic that needs to die, though I would be delighted to be shown otherwise :smile: 

```
// Firefox puts an inner border on buttons
button::-moz-focus-inner {
  border: 0;
}
```

I find myself including this in every project. I think others would also find it useful. @necolas if you are amiable I can put together a PR.
